packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/Phone/ConfirmRecoveryCode/container.tsx

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import ConfirmRecoveryCode from '.';
+import { ResendStatus } from '../../../../../lib/types';
+
+const ConfirmRecoveryCodeContainer = (_: RouteComponentProps) => {
+  const [errorMessage, setErrorMessage] = useState('');
+  const [resendErrorMessage, setResendErrorMessage] = useState('');
+  const [resendStatus, setResendStatus] = useState<ResendStatus>(
+    ResendStatus.none
+  );
+
+  const verifyCode = async (otpCode: string) => {};
+  const resendCode = async () => {};
+
+  const clearBanners = () => {
+    setErrorMessage('');
+    setResendErrorMessage('');
+    setResendStatus(ResendStatus.none);
+  };
+
+  // TODO: get from api
+  // const account = useAccount();
```

Comment:
Since this is in signin and not account settings, we should _not_ use `useAccount` since it attempts to query all account data. We have a ticket to remove it from reset PW pages and shouldn't have it anywhere else.